database/config.py

```diff
@@ -0,0 +1,6 @@
+DB = 'sqlite'
+#USER = ''
+#PASSWORD = ''
+#HOST = ''
+#PORT = ''
+DATABASE = 'nettacker.db'
```

Comment:
Please insert database values in json/dict in [core/config.py](https://github.com/zdresearch/OWASP-Nettacker/blob/master/core/config.py)  with default values. then import the config.py in here and use it to read the config. e.g.
```python
from core.config import _database_config
USER = _database_config()["USERNAME"]
```

Reply:
Sure, I will do that.